components/fieldgroups/intent/intent-details.schema.json

```diff
@@ -0,0 +1,132 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/intent-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "XDM Business Intent Details",
+  "type": "object",
+  "meta:tags": {
+    "b2bSchema": true
+  },
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/intent"],
+  "description": "Captures and maintains additional intent attributes.",
+  "definitions": {
+    "intent-details": {
+      "properties": {
+        "xdm:sourceSystemType": {
+          "title": "Source System Type",
+          "description": "Adobe identifier for the partner.",
+          "enum": ["BO", "ZI", "DE", "LS"],
+          "meta:enum": {
+            "BO": "Bombora",
+            "ZI": "Zoominfo",
+            "DE": "Demandbase",
+            "LS": "Leadspace"
+          }
+        },
+        "xdm:domain": {
+          "title": "Domain",
+          "description": "Account Domain.",
+          "type": "string"
+        },
+        "xdm:topicName": {
+          "title": "Topic Name",
+          "description": "Name of the topic.",
+          "type": "string"
+        },
+        "xdm:thirdPartyAccountID": {
+          "title": "Third Party Account ID",
+          "description": "Account ID in third party system.",
+          "type": "string"
+        },
+        "xdm:accountKey": {
+          "title": "Account Key",
+          "description": "Unique composite identifier of the account this intent is associated with.",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/b2b-source"
+        },
+        "xdm:topicID": {
+          "title": "Topic ID",
+          "description": "Unique ID for Topic.",
+          "type": "string"
+        },
+        "xdm:categoryName": {
+          "title": "Category Name",
+          "description": "Name of the Category, Keyword set to which topic belongs.",
+          "type": "string"
+        },
+        "xdm:keywordSetId": {
+          "title": "Keyword Set ID",
+          "description": "Unique ID of the keyword set.",
+          "type": "string"
+        },
+        "xdm:scoreString": {
+          "title": "Score String",
+          "description": "Intent Score (High, Med, Low).",
+          "type": "string"
+        },
+        "xdm:scoreNumber": {
+          "title": "Score Number",
+          "description": "Intent Score numeric.",
+          "type": "number"
```

Comment:
Type "number" maps to type "Double" in Java. For score, do you see a floating point value? Or would type "integer" with min and max value constraints would be more appropriate for your use case?

Reply:
Updated type to "integer"